recipe/meta.yaml

```diff
@@ -1,16 +1,17 @@
 {% set name = "openssl" %}
-{% set version = "1.1.1m" %}
+{% set version = "1.1.1n" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: http://www.openssl.org/source/{{ name }}-{{ version }}.tar.gz
-  sha256: f89199be8b23ca45fc7cb9f1d8d3ee67312318286ad030f5316aca6462db6c96
+  url: https://www.openssl.org/source/{{ name }}-{{ version }}.tar.gz
+  sha256: 40dceb51a4f6a5275bde0e6bf20ef4b91bfc32ed57c0552e2e8e15463372b17a
   patches:
-    - 0001-Don-t-use-USE_BCRYPTGENRANDOM-for-VS-older-than-2015.patch
+    # backport https://github.com/openssl/openssl/pull/17902
+    - patches/gh17902.patch
```

Comment:
This backport seems to be from the OpenSSL 3.x branch, so I don't think we need to adopt it for our OpenSSL 1.1.1* build.